CONTRIBUTING.rst

```diff
@@ -18,12 +18,37 @@ first in an appropriate venue, drafting a PEP and gathering feedback, and
 developing at least a prototype reference implementation of your idea.
 
 
+Contributing changes to existing PEPs
+-------------------------------------
+
+In general, most non-Draft/Active PEPs are considered to be historical
+documents rather than living specifications or documentation. Major changes to
+their core content usually require a new PEP, while smaller modifications may
+or may not be appropriate, depending on the PEP's status. See `PEP 1
+<https://www.python.org/dev/peps/pep-0001/#pep-maintenance>`__ for more.
+
+Copyediting and proofreading Draft and Active PEPs (including fixing spelling
+and formatting issues) is welcomed, and can be done via pull request to this
```

Comment:
I think the author should be allowed to have total control. In the end the PEP repo is a convenient place where authors place their proposals to be discussed by the community. Authors may appreciate help, but PEPs, unlike general documentation, *have* authors, and the process is geared towards their needs.

Reply:
I originally included an explicit conditional on this line, "at the discretion of the author(s)", but eliminated it in response to the request to make the more concise. I added back appropriate caveats both places this is referred to.

(As a sidenote, there are _some_ limits to what is an author's discretion over a PEP editor, when push comes to shove—insensitive or inappropriate language, syntax/technical issues, major breaches of PEP 1/12, etc. But not need to delve into that here.)